proto/system_config.proto

```diff
@@ -315,4 +315,28 @@ message CloudConfig {
 
   // Bridge port.
   int32 mqtt_bridge_port = 9;
-}
\ No newline at end of file
+}
+
+/*
+ * Configuration for device coupler
+ */
+message DeviceCouplerConfig {
+  // Name of trunk interface that connects to access switch
+  string trunk_iface = 1;
+
+  // Name of OVS bridge where device MACs from access s/w are learnt
+  string bridge = 2;
+
+  // gRPC target IP
+  string target_ip = 3;
```

Comment:
Trevor will probably say "target" is ambiguous.

Reply:
Changed to daq_grpc_ip